willsCheatMenu/src/hacks/inventory.ts

```diff
@@ -18,7 +18,7 @@ const itemify = (item: Item[], amount: number) =>
 new Hack(category.inventory, "Item stacker").setClick(async () => {
 	const num = await NumberInput.fire("Amount", "How many of every item would you like?", "question");
 	if (!num.value) return;
-	if (!(await Confirm.fire("Are you sure you want to get all items in the game?"))) return;
+	if (!(await Confirm.fire("Are you sure you want to get all items in the game?")).value) return;
```

Comment:
also made a inconsequential thing that keeps the item stacker from triggering even if the user denies the conf

Reply:
cus the item stacker was triggering even if you cancel the confirmation